GooglePlacesAutocomplete.js

```diff
@@ -556,18 +606,39 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
         setStateText(props.preProcess(text));
       }
 
-      request.open(
-        'GET',
-        `${url}/place/autocomplete/json?input=` +
-          encodeURIComponent(text) +
-          '&' +
-          Qs.stringify(props.query),
-      );
+      if (props.isNewPlacesAPI) {
+        const keyQueryParam = props.query.key
+          ? '?' +
+            Qs.stringify({
+              key: props.query.key,
+            })
+          : '';
+        request.open('POST', `${url}/v1/places:autocomplete${keyQueryParam}`);
+      } else {
+        request.open(
+          'GET',
+          `${url}/place/autocomplete/json?input=` +
+            encodeURIComponent(text) +
+            '&' +
+            Qs.stringify(props.query),
+        );
+      }
 
       request.withCredentials = requestShouldUseWithCredentials();
       setRequestHeaders(request, getRequestHeaders(props.requestUrl));
 
-      request.send();
+      if (props.isNewPlacesAPI) {
+        const { key, locationbias, types, ...rest } = props.query;
```

Comment:
Is it necessary to maintain `locaitonbias` in here?

Reply:
`locationbias` is for old Places API and can be stripped whereas `locationBias` is what is expected in new Places API. We maintain `locationBias` for new Places API.

Reply:
Ah got it, thank you for explaining 👍